client/src/components/settings/email.tsx

```diff
@@ -159,11 +159,7 @@ function EmailSettings({
       {isEmailVerified ? null : (
         <FullWidthRow>
           <HelpBlock>
-            <Alert
-              bsStyle='info'
-              className='text-center'
-              closeLabel={t('buttons.close')}
```

Comment:
When I tested this on production `closeLebal` wasn't doing anything that I noticed.